.github/workflows/how-to-run-workflows-locally.md

```diff
@@ -0,0 +1,14 @@
+# Run GitHub Actions on Your Machine
```

Comment:
How would one find this `how-to-run-workflows-locally.md` file? 
Is it meant to be be main readme file for the `workflows` section? 
If so, then maybe we just name it `README.md`.

That way you will automatically see the docuemantation when you naviagte to the workflows-folder. 

What do you think?